R_D_HW_5/homework5(2).py

```diff
@@ -0,0 +1,5 @@
+import time
+
+while True:
+    print("4.2 seconds")
```

Comment:
The task was `print "I love Python" every 4.2 seconds` but you use here some other prase.